### DISCUSSION TITLE
Error in fast_quantization

### DISCUSSION THREAD

**Discussion by CrHvG** [r-lidar/lidR#459] [General]
I get the following error :

Error in fast_quantization(las@data[["Z"]], zscale, zoffset) : 
  Non quantizable value outside the range of representable values of type 'int'

The only other mention I see on Stack Exchange references a bug that was fixed with 3.1.2, but I am using 3.1.4 and still get an error.

Here is my code (las attached):

tmp_las <- readLAS(...xxx...)
dtm = grid_terrain(tmp_las, algorithm = knnidw())
las_norm<-normalize_height(tmp_las, dtm)

[NIWO_2020_447000_4429000.zip](https://github.com/Jean-Romain/lidR/files/6971438/NIWO_2020_447000_4429000.zip)

**Comment by Jean-Romain** [comment]
See https://github.com/Jean-Romain/lidR/issues/460
